src/javaRosa.js

```diff
@@ -576,7 +576,12 @@ define([
 
         widget.save = function () {
             // override save to call out to rename itext
-            widget.mug.setItextId(widget.path, widget.getValue());
+            if (widget.mug.setItextId(widget.path, widget.getValue())) {
```

Comment:
Do you know who is calling this `save()` method? Typically [`widget.handleChange()` -> `updateValue()` -> `save()`](https://github.com/dimagi/Vellum/blob/save-button-fix/src/widgets.js#L61-L87), and somewhere in there the save button gets activated for most widgets. Do you know why the normal save button activation is not working for this widget? It's possible that this widget does not implement the widget protocol correctly, and you may be papering over a bigger problem with this fix.